psql_authnz.conf

```diff
@@ -8,5 +8,5 @@ respawn
 
 script
   . /opt/psql-authnz.env/bin/activate
-  exec su -s /bin/sh -c 'exec "$0" "$@"' postgres -- python /opt/psql-authnz/psql_authnz.py > /var/log/psql_authnz.log
+  exec su -s /bin/sh -c 'exec "$0" "$@"' postgres -- python /opt/psql-authnz/psql_authnz.py
```

Comment:
Yes, the environment variables are passed along, as long as they are EXPORTed.  I've tested this in Dev, and it seems to work at least!